### PR TITLE
fix: add border to quote CB

### PR DESCRIPTION
**Pull request by hofmann-felix**
Resolves: https://github.com/satellytes/satellytes.com/pull/216#issuecomment-981675981
Storybook: https://deploy-preview-226--satellytes-website-storybook.netlify.app/?path=/docs/components-quote

### What's included?
- A border was added to the Quote content block on the left side with `box-shadow`

### Preview
Bildschirmfoto 2021-11-29 um 15 42 05

### CODE REVIEW

**Comment by ghost** [comment]
# [Gatsby Cloud](https://www.gatsbyjs.com) Build Report
**satellytes.com-main**

:tada: Your build was successful! [See the Deploy preview here.](https://satellytescommain21751-fixquoteborder.gtsb.io)

## Build Details

[View the build logs here.](https://gatsbyjs.com/dashboard/f215130d-15e3-4176-9252-e13166fe48de/sites/674be368-adf0-4ac5-8fae-f9a4493adc51/builds/4c2b9679-bf8a-4ae2-9337-32bbee084f71/details)

:clock1: Build time: **7m**

## Performance

### Lighthouse report

| Metric | Score |
| ----- | ---- |
| Performance |   :large_orange_diamond: 71   |
| Accessibility |   :green_heart: 97   |
| Best Practices |   :green_heart: 100   |
| SEO |   :large_orange_diamond: 85   |

:link: [View full report](https://build-4c2b9679-bf8a-4ae2-9337-32bbee084f71.gtsb.io/reports/lighthouse/index.html)

**Review by milljoniaer — APPROVED** [review]
Looks good to me :+1:

File: src/components/quote/quote.tsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ const QuoteText = styled.blockquote`
   color: ${theme.palette.text.topline};
   padding-left: 20px;
   margin: 0;
+  box-shadow: inset 2px 0 0 ${theme.palette.text.topline};
```
Comment (fischeversenker) [inline src/components/quote/quote.tsx]: I wonder if this needs to be a `box-shadow`. Why not a simple `border-left`?
`box-shadow` has been known for it's bad performance on mobile devices... maybe that is outdated now but using a `box-shadow` to create a solid left border feels like it's overkill.

**Review by fischeversenker — APPROVED** [review]
I'm approving this to not block the PR and because it works perfectly fine. Nevertheless I'd suggest to try to always use the simplest possible solution that get's the job done. In that case - in my opinion - this would be a `border-left: ...` instead of `box-shadow: ...`. See my comment.

File: src/components/quote/quote.tsx (hunk 1 above)
Comment (georgiee) [reply src/components/quote/quote.tsx]: Figma does it like that so I guess false friend 😁 I suggest to use a border left too.

**Pull request merged**